resource-timing/resources/nested-contexts.js

```diff
@@ -0,0 +1,95 @@
+let destination = location;
+if (location.search == "?crossorigin") {
+    const host_info = get_host_info()
+    destination = location.protocol + "//" + host_info["REMOTE_HOST"] + ":" + location.port;
```

Comment:
As a followup, it might be good to have not-samesite tests here too, just to test the cross-process case.  hostinfo does not expose NOTSAMESITE_HOST at the moment, so that would need to get fixed too...  I filed https://github.com/web-platform-tests/wpt/issues/15819 on that.

Reply:
But this test is already cross-origin between localhost and some other host, or am I missing something?

Reply:
IIRC, @bzbarsky was asking for cross-origin and cross-site tests (so using hosts that have different domains)

Reply:
Right but this test is both cross-origin and cross-site if runs from localhost. Maybe what's missing is same-site cross-origin?

Reply:
This test does not normally run "from localhost".  It runs from one of the define test hosts in the test configuration.

Reply:
And for details, see the `get_host_info` function's implementation.

Reply:
Gotcha. I wasn't aware of these details. Thanks!